### DISCUSSION TITLE
Moved Entire Solution

### DISCUSSION THREAD

**Discussion by ToddForsberg** [oqtane/oqtane.framework#1190] [General] [answered]
After moving the entire solution to different folder, it stopped working because it is still looking for something in the old location.

Error occurs here in Program.cs in the server project:

`        public static IWebHost BuildWebHost(string[] args) =>
            WebHost.CreateDefaultBuilder(args)
                .UseConfiguration(new ConfigurationBuilder()
                    .AddCommandLine(args)
                    .Build())
                .UseStartup<Startup>()
                .ConfigureLocalizationSettings()
                .Build();
`
How can I correct this?

**Comment by leigh-pointer** [comment]
Have you referenced any external modules that you are working on?

**Reply by ToddForsberg** [reply]
No.  Not that I'm aware of.  I am new to Oqtane and was just working with Oqtane.Framework out of the box.

**Comment by hishamco** [comment]
> After moving the entire solution to different folder, it stopped working because it is still looking for something in the old location.

Why you moved the solution to another folder?

**Reply by ToddForsberg** [reply]
First folder was just temp.  Moved to another folder for continued testing and learning.

**Reply by hishamco** [reply]
Simply you can clone the repo and everything should work as expected

**Comment by ToddForsberg** [comment, marked as answer]
I think I got it working now.  

First was just doing a simple "Clean Solution" and rebuild.

Second, was editing the debug.cmd and release.cmd in "Modules" projects to reflect the new path for the solution.

**Reply by hishamco** [reply]
If you answered my above question, I will tell you there are some paths need to be modified in the cmd files ;)
